src/goto-analyzer/static_verifier.cpp

```diff
@@ -143,10 +143,8 @@ bool static_verifier(
     m.status() << "Writing XML report" << messaget::eom;
     out << xml_result;
   }
-  else
+  else if(options.get_bool_option("text"))
```

Comment:
Is this still being set anywhere?

Reply:
Looks obsolete now

Reply:
Is set with command line option --text, as before.

Reply:
Oh, it's just no longer the default. Ok!